roles/reproducer/tasks/configure_crc.yml

```diff
@@ -25,6 +25,24 @@
         never_default4: true
         state: present
 
+    - name: Ensure NetworkManager does not override /etc/resolv.conf
+      become: true
+      ansible.builtin.copy:
+        dest: "/etc/NetworkManager/conf.d/no-resolvconf.conf"
+        mode: "0644"
+        content: |
+          [main]
+          rc-manager = unmanaged
```

Comment:
had to go find that https://www.networkmanager.dev/docs/api/latest/NetworkManager.conf.html 

seems legit ;) and yeah I don't think we want NetworkManager doing anything to resolv.conf!

```
Set the resolv.conf management mode. This option is about how NetworkManager writes to /etc/resolv.conf, if at all. The default value depends on NetworkManager build options, and this version of NetworkManager was build with a default of "auto". Regardless of this setting, NetworkManager will always write its version of resolv.conf to its runtime state directory as /run/NetworkManager/resolv.conf. 
...

resolvconf: NetworkManager will run resolvconf to update the DNS configuration.

netconfig: NetworkManager will run netconfig to update the DNS configuration.

unmanaged: don't touch /etc/resolv.conf.

none: deprecated alias for symlink.

```